server/static/static.go

```diff
@@ -43,7 +45,7 @@ type StaticFileServer struct {
 
 // NewStaticFileServer returns a new static file server that will serve the
```

Comment:
we should probably just rename this package from static -> app or pull the app stuff out into its own package

Reply:
i'd like to do this for all static resources (svgs, css files, images, etc), just starting with the app js bundle because it's the most pressing